tests/admin_utils/test_logentry.py

```diff
@@ -320,28 +408,29 @@ def test_action_flag_choices(self):
                 self.assertEqual(log.get_action_flag_display(), display_name)
 
     def test_hook_get_log_entries(self):
-        LogEntry.objects.log_action(
+        LogEntry.objects.log_actions(
             self.user.pk,
-            ContentType.objects.get_for_model(Article).pk,
-            self.a1.pk,
-            "Article changed",
+            [self.a1],
             CHANGE,
             change_message="Article changed message",
+            single_object=True,
         )
         c1 = Car.objects.create()
-        LogEntry.objects.log_action(
+        LogEntry.objects.log_actions(
             self.user.pk,
-            ContentType.objects.get_for_model(Car).pk,
-            c1.pk,
-            "Car created",
+            [c1],
             ADDITION,
             change_message="Car created message",
+            single_object=True,
         )
+        exp_str_article = escape(str(self.a1)[:200])
+        exp_str_car = escape(str(c1)[:200])
+
         response = self.client.get(reverse("admin:index"))
-        self.assertContains(response, "Article changed")
-        self.assertContains(response, "Car created")
+        self.assertContains(response, exp_str_article)
+        self.assertContains(response, exp_str_car)
 
         # site "custom_admin" only renders log entries of registered models
         response = self.client.get(reverse("custom_admin:index"))
-        self.assertContains(response, "Article changed")
-        self.assertNotContains(response, "Car created")
+        self.assertContains(response, exp_str_article)
+        self.assertNotContains(response, exp_str_car)
```

Comment:
We should revert this unrelated refactoring.

Reply:
This is needed for the same reason I have mentioned [here](https://github.com/django/django/pull/16792/files#r1364857688) . Earlier the hardcoded string `"Car created"` was sent as `object_repr`, but now the `object_repr = str(obj)[:200]` is being generated dynamically. I am halting making the suggested changes for now, let's arrive at a consensus about the `object_repr`, in case we can find a more backward compatible way. Let me know what do you think about this..